generate/normalize_benchmark_config.sh

```diff
@@ -40,4 +40,4 @@ while read config; do
 			continue
 		fi
 	done
-done <../generate/benchmark_config.txt
+done <examples/benchmarks/config_"$BACKEND".txt
```

Comment:
Nitpick: As a shell noob, I'd think that this would be somehow better as the version in `generate/benchmarks.sh`, i.e.
```suggestion
done <"examples/benchmarks/config_$BACKEND.txt"
```
Is it equivalent?